pkg/test/framework.go

```diff
@@ -40,36 +40,46 @@ import (
 )
 
 var (
-	// Global framework struct
+	// Global is a global framework struct that the test can use.
 	Global *Framework
 	// mutex for AddToFrameworkScheme
 	mutex = sync.Mutex{}
-	// whether to run tests in a single namespace
-	singleNamespace *bool
+	// SingleNamespace determines whether tests are to be run in a single namespace or not.
+	SingleNamespace *bool
```

Comment:
Are users expected to set this?

Reply:
That's what I was initially thinking. It'd be better to just have that as an argument for `Setup` though, so I'll change that and unexport this field